src/main/java/com/wire/bots/hold/MessageHandler.java

```diff
@@ -26,132 +27,138 @@ public class MessageHandler extends MessageHandlerBase {
     public void onPhotoPreview(WireClient client, PhotoPreviewMessage msg) {
         UUID eventId = msg.getEventId();
-        UUID convId = msg.getConversationId();
+        QualifiedId conversationId = msg.getConversationId();
         UUID userId = client.getId();
         String type = Const.CONVERSATION_OTR_MESSAGE_ADD_IMAGE_PREVIEW;
 
-        persist(eventId, convId, userId, type, msg);
+        persist(eventId, conversationId.id, userId, type, msg);
 
         assetsDAO.insert(msg.getMessageId(), msg.getMimeType());
     }
 
     @Override
     public void onFilePreview(WireClient client, FilePreviewMessage msg) {
         UUID eventId = msg.getEventId();
-        UUID convId = msg.getConversationId();
+        QualifiedId conversationId = msg.getConversationId();
         UUID userId = client.getId();
         String type = Const.CONVERSATION_OTR_MESSAGE_ADD_FILE_PREVIEW;
 
-        persist(eventId, convId, userId, type, msg);
+        persist(eventId, conversationId.id, userId, type, msg);
 
         assetsDAO.insert(msg.getMessageId(), msg.getMimeType());
     }
 
     @Override
     public void onAudioPreview(WireClient client, AudioPreviewMessage msg) {
         UUID eventId = msg.getEventId();
-        UUID convId = msg.getConversationId();
+        QualifiedId conversationId = msg.getConversationId();
         UUID userId = client.getId();
         String type = Const.CONVERSATION_OTR_MESSAGE_ADD_AUDIO_PREVIEW;
 
-        persist(eventId, convId, userId, type, msg);
+        persist(eventId, conversationId.id, userId, type, msg);
 
         assetsDAO.insert(msg.getMessageId(), msg.getMimeType());
     }
 
     @Override
     public void onVideoPreview(WireClient client, VideoPreviewMessage msg) {
         UUID eventId = msg.getEventId();
-        UUID convId = msg.getConversationId();
+        QualifiedId conversationId = msg.getConversationId();
         UUID userId = client.getId();
         String type = Const.CONVERSATION_OTR_MESSAGE_ADD_VIDEO_PREVIEW;
 
-        persist(eventId, convId, userId, type, msg);
+        persist(eventId, conversationId.id, userId, type, msg);
 
         assetsDAO.insert(msg.getMessageId(), msg.getMimeType());
     }
 
     @Override
     public void onAssetData(WireClient client, RemoteMessage msg) {
         UUID eventId = msg.getEventId();
-        UUID convId = msg.getConversationId();
+        QualifiedId conversationId = msg.getConversationId();
         UUID userId = client.getId();
         String type = Const.CONVERSATION_OTR_MESSAGE_ADD_ASSET_DATA;
 
-        persist(eventId, convId, userId, type, msg);
+        persist(eventId, conversationId.id, userId, type, msg);
 
         try {
-            final byte[] assetData = client.downloadAsset(msg.getAssetId(), msg.getAssetToken(), msg.getSha256(), msg.getOtrKey());
+            final byte[] assetData = client.downloadAsset(
+                msg.getAssetId(),
+                null, // TODO(WPB-11287): Change null to default domain
```

Comment:
Yes, the comment might be a bit misleading to what it needs to be done, but at least we have an overview of where it will need to be changed as well when we do the database migrations from UUID to QualifiedID (id, domain)